JSDemos/Demos/PivotGrid/ChartIntegration/Angular/description.md

```diff
@@ -0,0 +1,44 @@
+
+If you use only the first argument and omit **integrationOptions**, default settings are used: row field values form chart [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/), and column field values form [arguments](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/). The Chart displays values of each data field on a separate [value axis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/). If you need to customize the resulting chart, specify the **integrationOptions** argument.  
+
+## Convert PivotGrid Fields to Chart Series
+You can use the following **integrationOptions** to specify how PivotGrid fields should be converted to chart series:
+
+- **alternateDataFields**: Boolean       
+Specifies whether to alternate data fields in the resulting chart. This demo sets the **alternateDataFields** field to **false**.
+
+- **dataFieldsDisplayMode**: *"singleAxis"* | *"splitAxes"* | *"splitPanes"*     
+Accepts one of the following values.   
+
+    - *"singleAxis"* - values of all data fields are displayed on a single [value axis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/).  
+    - *"splitAxes"* - each data field gets an individual value axis. Applies only if **putDataFieldsInto** is *"series"*.  
+    - *"splitPanes"* - each data field gets an individual [pane](/Documentation/ApiReference/UI_Components/dxChart/Configuration/panes/). Applies only if **putDataFieldsInto** is *"series"*.    
+
+    Applies only if the PivotGrid contains several [data fields](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/). In this demo, the **dataFieldsDisplayMode** property's value is *"splitPanes"*.
+
+- **inverted**: Boolean      
+If **true**, [row field](/Documentation/Guide/UI_Components/PivotGrid/Visual_Elements/#Headers) values go to the [chart arguments](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/) and [column field](/Documentation/Guide/UI_Components/PivotGrid/Visual_Elements/#Headers) values form [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/). If **false** - vice versa.
+
+- **putDataFieldsInto**: *"series"* | *"args"*     
+Accepts one of the following values.
+
+    - *"args"* - data fields go to the Chart arguments.  
+    - *"series"* - data fields go to the Chart series.
+
+## Advanced Customization
+The **integrationOptions** object also can contain the following functions that allow you to process PivotGrid data before visualization and customize the Chart and its series:
+
+- **processCell**       
+Processes a PivotGrid cell's data before the Chart displays it.
```

Comment:
How one can "process" a cell's data? Of which actions do you speak? Like "apply a formatting pattern" or something else that can't even imagine? The verb "to process" is very general by its nature, it could mean almost anything and conveys nothing in particular at the same time.

Reply:
That's the point. processCell allows customers to do almost anything with a cell's data.

Reply:
Alright, can it modify values or the "process" is limited to mostly cosmetic changes? As for me, I'd like to see a couple of examples in this line.

Reply:
There are two main problems with this suggestion that I discussed with @RomanTsukanov .
- This description is already huge and examples will definetely overload it.
- The more important fact is that this function allows developers to modify data in any way they want (not only cosmetic changes are possible). There aren't any particular examples since this function is designed to give developers complete freedom.

Reply:
This means you all set then. I will not insist.